src/Features/OnboardingTasks/ExtendedTask.php

```diff
@@ -0,0 +1,166 @@
+<?php
+/**
+ * A temporary class for creating tasks on the fly.
+ */
+
+namespace Automattic\WooCommerce\Admin\Features\OnboardingTasks;
+
+/**
+ * ExtendedTask class.
+ */
+class ExtendedTask extends Task {
```

Comment:
I'm a bit mixed on this long-term (works fine for a temporary fix until we deprecate old tasks).  

Should we keep this around for quick task creation and update the name to something like `GenericTask`?  I'm thinking not and we should encourage extended the `Task` class directly.

Reply:
The `TestTask` class duplicates almost all of this logic so it would be beneficial from that standpoint, but we could keep this around purely for testing if we decide to encourage use of the class.

Reply:
So this class is only used for adding the extended tasks coming from the client, the ones that will be deprecated?

In that case I would also say to keep it short-term, and maybe make the comment a little more explicit, that's its used for the deprecated tasks on the front-end.
Unless we also set the name to something like `OldExtendedTasks`? or `DeprecatedExtendedTasks`.